tests/e2e/test_rosacli_ingress.go

```diff
@@ -178,6 +179,39 @@ var _ = Describe("Edit default ingress",
 				ingress, _ = defaultIngress(*ingressList)
 				Expect(ingress.LBType).Should(ContainSubstring(ingress.LBType))
 			})
+
+		It("can customize ingress controller at install - [id:65798]",
+			labels.High,
+			labels.Day1Post,
+			labels.NonHCPCluster,
+			func() {
+				profile := PH.LoadProfileYamlFileByENV()
+				if !profile.ClusterConfig.IngressCustomized {
+					Skip("The ingress must be customized at install")
+				}
+				output, err := ingressService.ListIngress(clusterID)
+				Expect(err).ToNot(HaveOccurred())
+
+				ingressList, err := ingressService.ReflectIngressList(output)
+				Expect(err).ToNot(HaveOccurred())
+				defaultIngress := func(ingressList rosacli.IngressList) (*rosacli.Ingress, bool) {
+					for _, ingress := range ingressList.Ingresses {
+						if ingress.Default == "yes" {
+							return &ingress, true
+						}
+					}
+					return nil, false
+				}
+
+				ingress, _ := defaultIngress(*ingressList)
+				Expect(ingress.ExcludeNamespace).Should(ContainSubstring("test-ns1"))
```

Comment:
There shouldn't be hardcode here. You have to read the config file and verify it is expected to equal the value in config file

Reply:
@xueli181114 I believe I fixed it the way you recommended, can you take another look?